explorer/src/components/account/nftoken-hooks.tsx

```diff
@@ -0,0 +1,39 @@
+import { NftokenFetcher, NftokenTypes } from "@glow-xyz/nftoken-js";
+import useSWR, { SWRResponse } from "swr";
```

Comment:
I know that elsewhere in this review I left notes on how to use SWR with a stable fetcher that caches data on the basis of your _full_ inputs, but I'm also a little concerned about this being the first and only use of SWR in Explorer (ie. adding another 4K library just to cache this _one_ set of models) when you _could_ implement it just with `useEffect`/`useState`.

If you decide to continue with SWR here, can you:

* Use the `suspense: true` property of SWR, and wrap components that use this hook in `<React.Suspense>` with a sensible fallback. See `useAnchorProgram` for how we use this pattern in Explorer!
* Make the key/fetcher changes that I suggested elsewhere in this PR.

Reply:
I think SWR is nice and I'd like to use it in the future if I do more contributions to the library. I find the `useFetchAccountInfo` model pretty confusing and think it would be easier to move that cacheing to SWR. 

I'll look into suspense.

Reply:
Hmm using suspense makes the code a bit more awkward since then I have to pull out the scaffold for the NFT grid into a different component. What's the advantage of using suspense? We aren't doing waterfall loading.

Reply:
I can go on for hours about the benefits of suspense, but I'll stop short by just noting that with Suspense, you won't see a flash of ‘Loading...’ content if the data fetch is fast enough. With the current implementation you will _always_ cycle through `null` before getting actual data, which causes the UI to render twice and the layout to shift.

Reply:
Hmm, I don't think that's true with SWR since if the key is in the SWR cache it will return `data` on first render. I did a test for this and validated that I don't see Loading. I'll record a short video.

Reply:
Ah it looks like the SWR cache isn't set up to persist across renders. I forgot about this since in Glow / Luma we usually seed the cache with a server side render. I can look deeper at Suspense...

Reply:
For sure. I was never talking about subsequent render performance. This is a solution for _first_ render performance. Read more on fetch-as-you-render vs. render-then-fetch! https://17.reactjs.org/docs/concurrent-mode-suspense.html#traditional-approaches-vs-suspense

Reply:
👍 will read more. We usually solve this problem with SSR.